src/SharpLearning.Backend.Cntk.Test/CntkRawMnistTest.cs

```diff
@@ -295,6 +295,16 @@ public static Function ActivationFunction(Variable input, Activation activation)
                 new int[] { poolW, poolH }, new int[] { strideW, strideH });
         }
 
+        public static Function Reshape(Variable layer, int[] targetShape)
```

Comment:
this function seems rather redundant given it adds little value

Reply:
Yes, it was just to have all the operators on one class. The same goes for softmax and a few other functions which simply forwards to the Cntk API.